pkg/render/common/networkpolicy/networkpolicy.go

```diff
@@ -279,6 +279,10 @@ func (h *NetworkPolicyHelper) ManagerSourceEntityRule() v3.EntityRule {
 	return CreateSourceEntityRule(h.namespace("tigera-manager"), "tigera-manager")
 }
 
+func (h *NetworkPolicyHelper) APIServerSourceEntityRule() v3.EntityRule {
```

Comment:
I think this function should have the `operatorv1.ProductVariant` passed in, similar to many of the funcs in apiserver.go . Otherwise, down the line, others may start using this function in the context of opensource and create a defect that way.